src/main/java/dekk/pw/pokemate/tasks/CatchPokemon.java

```diff
@@ -31,10 +31,11 @@ public void run(Context context) {
                 } else {
                     //find any pokeball we can.
                     for (Pokeball pb : Pokeball.values()) {
-                        Item item = context.getApi().getInventories().getItemBag().getItem(pb.getBallType());
-                        if (item != null && item.getCount() > 0)
+                        ball = context.getApi().getInventories().getItemBag().getItem(pb.getBallType());
+                        if (ball != null && ball.getCount() > 0){
                             pokeball = pb;
-                        break;
+							break;
```

Comment:
Next two lines are too far indented, it seems this repository uses four spaces per indentation.
The fix seems valid though.